type_data/migrations/25_fix_undertaker_bat_skin.rb

```diff
@@ -0,0 +1,4 @@
+migrate :skin, :furrier do
```

Comment:
looks good, is there a stale rule that should be deleted due to these fixes?

Reply:
I attempted to remove two stale rules, however the CI pipeline failed whenever I added in the removes.
Originally I included:
`migrate :skin, :furrier do
  delete(:name, %{leathery bat wings})
  delete(:name, %{agresh bear claw})
end`
But travis CI didn't seem to like that.

Reply:
No further input. Closing conversation and merging PR.